### DISCUSSION TITLE
Question on updating to 2.5.255

### DISCUSSION THREAD

**Discussion by fandd1234** [requarks/wiki#4779] [Help / Questions]
Hi all - I just tried to update the wiki,js from 2.5.219 to 2.5.255 but after I've follow the steps from this link <https://docs.requarks.io/install/upgrade>, I still couldn't verify it was updated to 2.5.255.

Is there anyone encountered the same please? and how can I verify if I've successfully updated to latest version apart from checking it on the web console? Thanks.

![Capture1](https://user-images.githubusercontent.com/92530680/145788411-a2d0838c-433d-4629-9b9e-9f8e14e0b02b.JPG)

**Comment by NGPixel** [comment]
Which platform are you using?

**Reply by fandd1234** [reply]
I'm using Linux (redhat 8.2), thank you

**Reply by NGPixel** [reply]
Did you restart Wiki.js afterwards? Are you using docker?

**Reply by fandd1234** [reply]
> Did you restart Wiki.js afterwards? Are you using docker?

Yes - I did restart wiki.js but still couldn't see the updated version were applied and that wasn't a docker.

Thank you.
